cvm/quantization/tfm_ops.py

```diff
@@ -164,6 +195,11 @@ def rewrite(self, op, **kwargs):
         op = self._matrix_decomposition(op, params, infer_shapes)
         return op
 
+
+    def compile(self, op, **kwargs):
+        childs = kwargs['childs']
+        attrs = kwargs['attr']
```

Comment:
debug or half-done code should be well-considered before commit and push, please delete it.